rollup/app/lwc/rollupForceRecalculation/rollupForceRecalculation.html

```diff
@@ -1,77 +1,93 @@
+            options={rollupMetadataOptions}
+            onchange={handleComboChange}
+          ></lightning-combobox>
+        </template>
+        <template if:false={isCMDTRecalc}>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="Calc Item SObject API Name"
+            name="calcItemSObjectName"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the rollup field on calc item"
+            name="opFieldOnCalcItem"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the lookup field on calc item"
+            name="lookupFieldOnCalcItem"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the lookup field on the lookup object"
+            name="lookupFieldOnLookupObject"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="API Name of the rollup field on the lookup object"
+            name="rollupFieldOnLookupObject"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="Lookup SObject API Name"
+            name="lookupSObjectName"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-input
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            type="text"
+            label="Rollup Operation Name (SUM/MIN/MAX/COUNT/COUNT_DISTINCT/CONCAT/CONCAT_DISTINCT/AVERAGE/FIRST/LAST)"
+            name="operationName"
+            onchange={handleChange}
+            required
+          >
+          </lightning-input>
+          <lightning-textarea
+            class="slds-col slds-form-element slds-form-element_horizontal"
+            onchange={handleChange}
+            label="Where Clause SOQL Filters (Optional)"
+            name="potentialWhereClause"
+          ></lightning-textarea>
+        </template>
         <lightning-button class="slds-col slds-m-top_small" onclick={handleSubmit} label="Start rollup!"></lightning-button>
```

Comment:
I haven't figured out why exactly it's failing, but the CMDT option fails in my scratch org with the error `There was an error performing your rollup: duplicate field selected: Id`. My rollup works fine from the UI (i.e., when I create a new `Contact` from the UI, it correctly rollups to the parent `Account` record), so it seems to be an issue just when running it from this component.

![image](https://user-images.githubusercontent.com/1267157/112763612-0e355f80-8fba-11eb-885e-cada3d793762.png)

Reply:
Here's a screenshot of the `Contact --> Account` CMDT rollup

![image](https://user-images.githubusercontent.com/1267157/112763873-248feb00-8fbb-11eb-8dd8-3a8a254fad43.png)

Reply:
Aaaand here's a screenshot from the debug log in dev console

![image](https://user-images.githubusercontent.com/1267157/112764021-aed84f00-8fbb-11eb-84ad-47eb52306335.png)

Reply:
I added an extra debug statement in `Rollup` to see the query string, here's the query string with the duplicate `Id` field

![image](https://user-images.githubusercontent.com/1267157/112764266-97e62c80-8fbc-11eb-89ee-f47ba47979a1.png)

Reply:
Ah I know what the issue is. My bad, I can get that fixed up. I'm adding the jest unit tests at the moment, let me get back to you about this stuff as well.